web/src/pages/Profile/index.tsx

```diff
@@ -0,0 +1,248 @@
+import Link from "next/link";
+import { SetStateAction, useState } from "react";
+import { MdAccountCircle, MdAddCircle } from "react-icons/md";
+
+import Button from "../../components/Inputs/Button";
+import LinkIconButton from "../../components/Inputs/LinkIconButton";
+import SearchBar from "../../components/Inputs/SearchBar";
+import ImageCard from "../../components/Surfaces/ImageCard";
+
+/*
+ *
+ * Note to viewer:
+ * Please bully, I must learn the error in my ways.
+ *
+ * */
+
+export default function Index() {
+  const [text, setText] = useState("");
+
+  function handleChange(value: string) {
+    setText(value);
+  }
+
+  const user_views = 1;
+  const user_saves = mockData.length;
+  const user_name = "username";
+  const user_icon = "https://avatars.githubusercontent.com/u/79925808?v=4";
+
+  return (
+    <>
+      {/* TOP BORDER */}
```

Comment:
I would remove all of these comments. They may seem useful, but comments have a lot of hidden costs. (Imagine the code is refactored, but the comments are mistakenly not. Now you have misleading comments that hinder reasoning.) Anyone who comes to edit this will just pull out inspect element anyway.